deploy/docker-compose/README.md

```diff
@@ -0,0 +1,390 @@
+
+- `hello-world.json `: Hello World!
+- `lung-seg.json`: AI Lung Segmentation MAP
+- `liver-seg.json`: AI Liver Segmentation MAP
+- `combo.json`: AI Lung & AI Liver Combo
+
+### Hello World
+
+#### Description
+
+This example uses the `alpine` image to print all files found in the input directory.
+
+1. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/hello-world.json"  http://localhost:5001/workflows
+
+   {"workflow_id":"849d4683-006b-410c-aa17-d0474ee26b7b"}
+   ```
+   If the `curl` command runs successfully, expect a `workflow_id` to be returned and printed to the terminal.
+1. Navigate to Orthanc, select any study and then click *Send to DICOM Modality* from the menu on the left.
+   In the popup dialog, select **MONAI-DEPLOY** to start a C-STORE request to the Informatics Gateway.
+1. To see the output from the container, run the following commands:
+   ```bash
+   $ docker container list -a | grep alpine
+   # locate the container ID and run the following
+   $ docker logs {CONTAINER ID}
+   # expect a list of files to be printed
+   /var/monai/input/1.2.826.0.1.3680043.2.1125.1.19616861412188316212577695277886020/1.2.826.0.1.3680043.2.1125.1.34918616334750294149839565085991567/1.2.826.0.1.3680043.2.1125.1.60545822758941849948931508930806372.dcm.json
+   /var/monai/input/1.2.826.0.1.3680043.2.1125.1.19616861412188316212577695277886020/1.2.826.0.1.3680043.2.1125.1.34918616334750294149839565085991567/1.2.826.0.1.3680043.2.1125.1.60545822758941849948931508930806372.dcm
+   ...
+   ```
+
+### AI Liver Segmentation MAP
+
+#### Description
+
+In this section, we will download a DICOM dataset, upload it to Orthanc and then run the [Liver Segmentation MAP](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/examples/apps/ai_livertumor_seg_app) from the
+[MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk). Finally, we can expect the AI-generated segmentation result to appear in Orthanc.
+
+1. Download the [Abdomen CT dataset](#running-a-monai-deploy-workflow) dataset
+2. Upload the dataset as described in [Uploading Data](#upload-dicom-datasets)
+3. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/liver-seg.json"  http://localhost:5001/workflows
+
+   {"workflow_id":"811620da-381f-4daa-854d-600948e67228"}
+   ```
+   If the `curl` command runs successfully, expect a `workflow_id` to be returned and printed to the terminal.
+4. Navigate to Orthanc, select any study and then click *Send to DICOM Modality* from the menu on the left.
+   In the popup dialog, select **MONAI-DEPLOY** to start a C-STORE request to the Informatics Gateway.
+5. Wait for the workflow to complete, reload the Orthanc study page and expect a new series to appear with the segmentation object.
```

Comment:
Please add an estimated time for this complete. How do users know it is done? Should the "State:Running" on the Orthanic page change to complete?

Reply:
Can you please specify that the new study will show up in the Patient studies?

Reply:
updated section